websites/E/Estácio/presence.ts

```diff
@@ -0,0 +1,306 @@
+          if (paths[0] === 'disciplinas'
+            && paths[2] === 'conteudos'
+            && paths[3]
+            && isValidUUID(paths[3])) {
+            const themeName = document.querySelector(
+              'header[data-testid="header-modal-conteudo"] p > span',
+            )?.textContent
+
+            if (themeName && showTheme) {
+              let state = removeThemePrefix(themeName)
+
+              if (iframeContent?.topic)
+                state += ` - ${iframeContent.topic}`
+
+              presenceData.state = state
+            }
+            else {
+              presenceData.state = 'Estudando...'
+            }
+          }
+        }
+      }
+
+      // Exercícios
+      if (document.location.pathname.includes('/exercicios')) {
+        const disciplineName = document.querySelector(
+          'main h1',
+        )?.textContent
+
+        if (disciplineName && showDisciplineName) {
+          presenceData.state = `Visualizando exercícios: ${disciplineName}`
+        }
+        else if (disciplineName && !showDisciplineName) {
+          presenceData.state = 'Visualizando exercícios'
+        }
+
+        if (!showVisualizingExercises)
+          presenceData.state = ''
+      }
+
+      // Trabalhos
+      if (document.location.pathname.includes('/trabalhos')) {
+        const disciplineName = document.querySelector(
+          'main h1',
+        )?.textContent
+
+        if (disciplineName && showDisciplineName) {
+          presenceData.state = `Visualizando trabalhos: ${disciplineName}`
+        }
+        else if (disciplineName && !showDisciplineName) {
+          presenceData.state = 'Visualizando trabalhos'
+        }
+
+        if (!showVisualizingWorks)
+          presenceData.state = ''
+      }
+    }
+
+    // Avaliações
+    if (document.location.pathname.includes('/avaliacoes')) {
+      presenceData.state = 'Visualisando avaliações'
+
+      if (!showVisualizingAvaliations)
+        presenceData.state = ''
+    }
+
+    // Formação socioemocional
+    if (document.location.pathname.includes('/formacao-socioemocional')) {
+      presenceData.state = 'Visualisando formação socioemocional'
+
+      if (!showVisualizingSocioemotionalFormation)
+        presenceData.state = ''
+    }
+
+    // Cursos complementares
+    if (document.location.pathname.includes('/cursos-complementares')) {
+      presenceData.state = 'Visualisando cursos complementares'
+
+      if (!showVisualizingComplementaryCourses)
+        presenceData.state = ''
+    }
+
+    // Calendário Acadêmico
+    if (document.location.pathname.includes('/calendarios')) {
+      presenceData.state = 'Visualisando o calendário acadêmico'
+
+      if (!showVisualizingAcademicCalendar)
+        presenceData.state = ''
+    }
+  }
+
+  // Campus
+  if (host === 'sia.estacio.br' && document.location.pathname.includes('/sianet')) {
+    presenceData.details = 'Navegando no campus virtual'
+
+    if (!showCampus)
+      presenceData.details = ''
+  }
+
```

Comment:
Rather than set the details to an empty string, `delete` the properties instead
```ts
// example
delete presenceData.details
delete presenceData.state
```